src/media/playback/playback_sensor.cpp

```diff
@@ -30,7 +30,7 @@ playback_sensor::playback_sensor(device_interface& parent_device, const device_s
     m_sensor_description(sensor_description),
     m_sensor_id(sensor_description.get_sensor_index()),
     m_parent_device(parent_device),
-    _default_queue_size(1)
+    _default_queue_size(2)
```

Comment:
As we said: let's take this back down to 1

Reply:
done